relay-statsd/src/lib.rs

```diff
@@ -247,7 +250,24 @@ pub fn init<A: ToSocketAddrs>(
     let socket = UdpSocket::bind("0.0.0.0:0").unwrap();
     socket.set_nonblocking(true).unwrap();
 
-    let statsd_client = if buffering {
+    let statsd_client = if aggregating {
+        let host = host.to_socket_addrs().unwrap().next().unwrap();
+        let statsdproxy_sink = StatsdProxyMetricSink::new(move || {
```

Comment:
How does this sink behave compared to our buffered config, I assume the aggregator also immediately accepts, aggregates in the background and then uses a buffered UDP sink? Depending on how the aggregator flushes a buffered UDP sink could still be beneficial.

Reply:
The `Upstream` itself is not buffered though, right?

https://github.com/getsentry/statsdproxy/blob/0b6d207423ad979b571b3807837142ce3cebaeb8/src/middleware/upstream.rs#L22-L35